public/app/plugins/panel/timeseries/models.gen.ts

```diff
@@ -0,0 +1,16 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// This file was autogenerated by cuetsy. DO NOT EDIT!
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+import * as ui from '@grafana/schema';
+
+export const modelVersion = Object.freeze([0, 0]);
+
+
+export interface PanelOptions {
```

Comment:
🤔 

I guess this is the same as:
```
export interface TimeSeriesOptions extends OptionsWithLegend, OptionsWithTooltip {}
```

if we add it here... lets remove it from Types... otherwise it is confusing to have duplicates :)

Reply:
Yup, absolutely - this is exactly the kind of adjustments that @grafana/user-essentials folks made while we were mobbing theirs in #49479 and #49580